plugins/linux-v4l2/data/locale/en-US.ini

```diff
@@ -13,3 +13,4 @@ ColorRange.Default="Default"
 ColorRange.Partial="Partial"
 ColorRange.Full="Full"
 CameraCtrls="Camera Controls"
+CloseOnHide="Close device when not visible"
```

Comment:
In win-dshow, We use the string "Deactivate when not showing". I think it makes sense to use the same string here.

Additionally, you should only enclude the English text. Other languages are fetched using Crowdin.